packages/kit/src/runtime/client/fetcher.js

```diff
@@ -86,7 +95,14 @@ export function initial_fetch(resource, opts) {
 		return Promise.resolve(new Response(body, init));
 	}
 
-	return native_fetch(resource, opts);
+	const patchedOpts = DEV
+		? {
+				...opts,
+				__sveltekit_fetch__: true
+		  }
+		: opts;
+
```

Comment:
Arguably, this isn't super nice but it allows us to let the patched `window.fetch` know that it was called via `event.fetch`, therefore allowing us to always call `window.fetch`.